src/http/Request.ts

```diff
@@ -1,27 +1,33 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the MIT License.
 
-import { HttpMethod, HttpRequest } from '@azure/functions';
+import { HttpMethod, HttpRequest, HttpRequestHeaders, HttpRequestParams, HttpRequestQuery } from '@azure/functions';
+import { AzureFunctionsRpcMessages as rpc } from '../../azure-functions-language-worker-protobuf/src/rpc';
+import { fromTypedData } from '../converters/RpcConverters';
+import { fromNullableMapping, fromRpcHttpBody } from '../converters/RpcHttpConverters';
 
-export class RequestProperties implements HttpRequest {
-    method: HttpMethod | null = null;
-    url = '';
-    originalUrl = '';
-    headers: { [key: string]: string } = {};
-    query: { [key: string]: string } = {};
-    params: { [key: string]: string } = {};
-    body?: any;
-    rawBody?: any;
-    [key: string]: any;

```

Comment:
Is there a reason this part is left out now?

Reply:
All it's saying is that the type for any other properties not explicitly mentioned is `any`, but `any` could literally be anything so there's no point. It's actually pretty discouraged to ever use `any` in TypeScript (it kind of defeats the point of _Type_Script), but unfortunately this repo uses it in a lot of places. I plan to slowly clean it up, but it'll take a while

Reply:
It does also specify that the key type must be `string` though, right? Or is that always the case anyway?

Reply:
Here are a few examples showing the nuances

With `[key: string]: any;`, you basically never get errors. TypeScript lets you do anything:
<img width="355" alt="Screen Shot 2021-12-02 at 11 16 52 AM" src="https://user-images.githubusercontent.com/11282622/144488273-b6cb1640-36af-4b44-a642-e235bd26b953.png">

If I did `[key: string]: string;` instead, TypeScript assumes `value` is `string` regardless of the key type:
<img width="549" alt="Screen Shot 2021-12-02 at 11 17 34 AM" src="https://user-images.githubusercontent.com/11282622/144488552-2dd946b3-06d5-441d-96b4-aeebd88dac9d.png">

Lastly, if you get rid of `[key: string]: ...` TypeScript only enforces types if you access a property with a `.` (which is the most common way to access properties). This is the behavior I prefer. We should get an error unless we define the type for the `test` property.
<img width="539" alt="Screen Shot 2021-12-02 at 11 17 14 AM" src="https://user-images.githubusercontent.com/11282622/144488406-e73e588d-56a3-4533-a30c-f7c4a2536ff9.png">

Reply:
That makes sense, thanks!